.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please check your .gitignore file, the `.idea` files sholdn't be pushed to git. 
fix please in way that this file will be removed  from PR.

Reply:
gitignore file was rewritten

Reply:
after rewriting gitignore you should remove not needed files (like vcd.xml) from the git stage area. it should not be present in github repository.